airflow-core/src/airflow/models/connection.py

```diff
@@ -497,7 +497,7 @@ def get_connection_from_secrets(cls, conn_id: str) -> Connection:
                     SecretCache.save_connection_uri(conn_id, conn.get_uri())
                     return conn
             except Exception:
-                log.exception(
+                log.debug(
```

Comment:
By desing we want to allow looping through different secrets backend for Connection and not raise an error

Reply:
Details: https://github.com/apache/airflow/pull/50329/files#r2085330762